ciongke/test-image/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    curl \
+    python \
+    gcc \
+    g++ \
+    libc6-dev \
+    make \
+    ca-certificates \
+    && rm -rf /var/lib/apt/lists/*
+
+# gcloud
```

Comment:
I'm not sure it's worth optimizing that. The slow part is the `apt-get` calls in the first `RUN`. After that it's a quick build. `ENV` calls are pretty much free.